pkg/plugins/platform/templates.go

```diff
@@ -69,10 +69,8 @@ services:
     image: {{ .router_image }}:{{ .router_tag }}
     container_name: nginx-proxy
     ports:
-      - "80:80"
-      - {{ .mailhogport }}:{{ .mailhogport }}
-      - {{ .dbaport }}:{{ .dbaport }}
-      - {{ .dbport }}:{{ .dbport }}
+      {{ range $port := .ports }}- "{{ $port }}:{{ $port }}"
```

Comment:
Sometimes this is true, but I'm not sure it holds in general. For user-defined services it makes sense that there would be a 1:1 ratio, but for things like MariaDB/nginx do we really want these changed? If someone wants to change from port 80 on HTTP to port 25049, do we really want to remap that within the pod as well? I'm not sold on that being a great idea.